crates/curp/src/client/tests.rs

```diff
@@ -541,6 +542,68 @@ async fn test_retry_propose_return_retry_error() {
     }
 }
 
+#[traced_test]
+#[tokio::test]
+async fn test_read_index_success() {
+    let connects = init_mocked_connects(5, |id, conn| {
+        conn.expect_propose_stream()
+            .return_once(move |_req, _token, _timeout| {
+                assert_eq!(id, 0, "followers should not receive propose");
+                let resp = async_stream::stream! {
+                    yield Ok(build_propose_response(false));
+                    yield Ok(build_synced_response());
```

Comment:
definition of this two `build_*_response` are not found?

Reply:
Fixed.